### ISSUE TITLE
Meetings: Bi-weekly VC Call to Discuss Open Issue and Progress

### ISSUE THREAD

**Issue by eriktaubeneck** [patcg-individual-drafts/ipa#47]
We will be holding bi-weekly calls to discuss and make progress on open issues on IPA, open to anyone participating in the PATCG (which is, in turn, is an open community group.) We will take minutes in the same manner as the PATCG meetings, and we will report important progress back in the regular PATCG meetings.

## Timing
The IPA authors and collaborators span many timezones, which makes timing difficult. For now, we are pegging the meeting to the second and fourth Wednesday 6am SGT (Singapore) and run for 1 hr. Our first meeting will be **March 21st / 22nd, 2023**.

For others, that start time translates to:

| Time  | Day | Location |
| ------| --- | -------- |
| 06:00 | Wed | Singapore |
| 07:00 | Wed | Tokyo |
| 08:00 | Wed | Sydney (AEST) |
| 09:00 | Wed | Sydney (AEDT) |
| 14:00 (2 PM) | Tue | Seattle PST |
| 15:00 (3 PM) | Tue | Seattle PDT |
| 17:00  (5 PM) | Tue | New York EST |
| 18:00 (6 PM) | Tue | New York EDT |
| 22:00 (10 PM) | Tue | London |
| 23:00 (11 PM) | Tue | London (BST) |
| 23:00 (11 PM) | Tue | Brussels |
| 00:00 | Wed | Brussels (ST) |

## Logistics
- The meeting will [take place in Zoom](https://fb.zoom.us/j/91485784740?pwd=TVdRQnYvTUxmb3QrUkN1UXB4c21NUT09)
- Minutes will be taken in [Google docs](https://docs.google.com/document/d/1AsQUVEka9vttOsVCnKYT8FIvn3rPvDpjHFxUpuHDm6E/edit?usp=sharing)
- Agenda Requests can be made by opening a [Biweekly Meeting Agenda Request Issue](https://github.com/patcg-individual-drafts/ipa/issues/new/choose).

**Comment by eriktaubeneck** [comment]
@bmayd yes, with a biweekly there's bound to be some conflicts. We're planning to attend both, and hopefully folks relevant to the topics (agenda coming soon) will attend.

**Comment by csharrison** [comment]
For convenience, here are all the time zones for this event:
https://everytimezone.com/s/6186ba54

**Comment by eriktaubeneck** [comment]
For clarity, this meeting is occasionally not bi-weekly, and will always be the 2nd and 4th Tuesday of the month. The next meeting is April 11th, 2023 (and not today, April 4th, 2023.)

**Comment by seanturner** [comment]
Would you like for me to add this to W3C Calendar? I can add it as something like PATCG/IPA.

**Comment by bmcase** [comment]
@seanturner, yes that sounds good to add to the W3C Calendar.

**Comment by seanturner** [comment]
@bmcase I have updated the W3C Calendar. Pleas double check that I entered them correctly.

If you need to update/cancel any of these let me know.

**Comment by bmcase** [comment]
We will cancel this week's (May 24) Bi-weekly IPA call.

**Comment by bmcase** [comment]
Since many of us are traveling for the London PATCG which is Wed and Thur this week, we'll go ahead and cancel this week's IPA biweekly call.  The two open agenda items in our repo are:
1.  [update on sharding designs](https://github.com/patcg-individual-drafts/ipa/issues/75) which will be discussed [Thursday in the PATCG](https://github.com/patcg/meetings/tree/main/2023/06/14-london)
2. a [Rust code walk through](https://github.com/patcg-individual-drafts/ipa/issues/76) which we'll plan to do on the next bi-weekly call in two weeks.

**Comment by eriktaubeneck** [comment]
Apologies for the late notice, but between vacations and lack of agenda items, we'll go ahead and cancel today's call (Aug 8th/9th).

If folks have agenda items they'd like to discuss before TPAC, please [open a new issue](https://github.com/patcg-individual-drafts/ipa/issues/new?assignees=eriktaubeneck&labels=agenda%2B&projects=&template=biweekly-meeting-agenda-request.md&title=IPA+Biweekly+Meeting+Agenda+Request+-+) and we'll discuss it on Aug 22nd/23rd.

**Comment by bmcase** [comment]
We will cancel this week's 22rd/23rd call again as we haven't had any new agenda items and some of us are still rotating through vacations.

**Comment by bmcase** [comment]
Apologies for the late notice, but we'll plan to keep today's call.  We will plan to have a working session with whoever wants to join on the steelmaning the comparison doc from TPAC with a focus on the IPA column.  Particularly there was some feedback to break out the security section into more cases.
Links:
original comparison doc https://docs.google.com/document/d/1oMe3Q7DMG3xzl-peIdq6voVD4PKD822-W4nf4ak7s2E/edit
steelman version https://docs.google.com/document/d/127HgUw8j5G59zOd0PmMZNXmhIETXHPu_Ub0_JIv0YV8/edit

**Comment by bmcase** [comment]
Since with have two PATCG virtual meetings week, we'll go ahead and cancel this week's IPA call.

**Comment by eriktaubeneck** [comment]
We are considering a couple changes to this meeting:

1. With Daylight Savings Time ending in the US, we are considering moving this meeting 1 hour later in Singapore, to make it a bit easier for them. For those in timezones that fell back, it would stay at the same time as before (e.g., 3pm PST.) Here is a link with all timezone: https://everytimezone.com/s/5fcdb20b
2. We're considering moving this from bi-weekly to monthly. We've had a less full agenda and cancelled as often as not, so this is likely a more appropriate cadence. We'd keep to the second Tuesday/Wednesday of the month.

Please comment here and/or in Slack if you have any objections, otherwise we'll confirm the changes by 11/13.

Also note, due to holidays, we'll be cancelling the 11/22 and 12/27 meetings.

**Comment by eriktaubeneck** [comment]
With no agenda items, we'll cancel this weeks IPA call on (11-7 / 11-8).

**Comment by eriktaubeneck** [comment]
Following up to [this comment](https://github.com/patcg-individual-drafts/ipa/issues/47?fbclid=IwAR1fN9LnLWikgGijY9zLy0Sj6TsJiLQHC0g37NkCBtznHopAdYmc2VQWImI#issuecomment-1795769069), we'll move this meeting to once a month, on the second Tuesday/Wednesday of the month. We will also anchor the meeting to 3pm PT (for both PST and PDT).

**Comment by martinthomson** [comment]
@eriktaubeneck, do you want to tweak the title and opening post to reflect that change?  This issue is pinned, but people coming in through the front door might not scroll to the bottom to find your update.

**Comment by eriktaubeneck** [comment]
Tomorrow (2024/03/12), we'll continue discussion on the sharded shuffle. (See [issue 972](https://github.com/private-attribution/ipa/issues/972) on the IPA implementation repo.) cc @akoshelev @marianapr @schoppmp @danielmasny

**Comment by eriktaubeneck** [comment]
The meeting on 2024/04/09 will be cancelled due to overlap with the [PATCG F2F meeting](https://github.com/patcg/meetings/tree/main/2024/04/09-boston).

**Comment by eriktaubeneck** [comment]
The meeting on 2024/06/11 is cancelled due to lack of agenda items. Please feel free to ping here with agenda items for future meetings!
